chaos_kitten/cli.py

```
@@ -201,6 +208,30 @@ def scan(
     try:
         import asyncio
         results = asyncio.run(orchestrator.run())
+        
+        if cors and target_url:
```

Ruff (0.15.1)

[error] Undefined name `target_url`

(F821)